tools/workflow/clean_aws_rsc.go

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package main
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/elbv2"
+	"log"
+	"strings"
+	"time"
+)
+
+const (
+	rsc_region               = "us-west-2"
+	containLbString          = "aoc-lb"
+	pastDayDelete            = 5
+	pastDayDeleteCalculation = -1 * time.Hour * 24 * pastDayDelete //Currently, deleting resources over 5 days
+)
+
+func main() {
+
+	log.Printf("Beging terminating EC2 Instances")
+	terminateEc2InstancesError := terminateEc2Instances()
+	if terminateEc2InstancesError != nil {
+		log.Fatalf("Exit terminating EC2 Instances")
+	}
+
+	log.Printf("Begin destroy Load Balancer resources")
+	destroyLoadBalancerError := destroyLoadBalancerResource()
+
+	if destroyLoadBalancerError != nil {
+		log.Fatalf("Exit destroy Load Balancer Resources")
+	}
+
+	log.Printf("Finish destroy AWS resources")
+}
+
+func terminateEc2Instances() error {
+	// set up aws go sdk ec2 client
+	testSession, err := session.NewSession(&aws.Config{
+		Region: aws.String(rsc_region)},
+	)
+
+	if err != nil {
+		log.Fatalf("Error creating session %v", err)
```

Comment:
I think you should pick one: either return the errors or exit immediately with log.Fatalf. Don't mix the two within the function.

Reply:
True. I fixed it in the latest commit but waiting for the PR build to success to use the cache.